pkg/actors/actors.go

```diff
@@ -1028,7 +1034,7 @@ func (a *actorsRuntime) RegisterInternalActor(ctx context.Context, actorType str
 		log.Debugf("Registering internal actor type: %s", actorType)
 		actor.SetActorRuntime(a)
 		a.actorsConfig.Config.HostedActorTypes.AddActorType(actorType, actorIdleTimeout)
-		if a.placement != nil {
+		if a.placementEnabled {
```

Comment:
if placementEnabled is true but if placement is not initialised properly, can it be an issue here?

Reply:
I don't think it should be. Before, `a.placement` was `nil` if placement was disabled. Now, it could be non-nil, but `a.placementEnabled` will be false. See: https://github.com/dapr/dapr/pull/7281/files#diff-aeb183ea3b81464cb579aa3e309be6bbfa88d6d900cfdbddb8c74d89ee7d4dc6L242-L243